tests/test_html2htpy.py

```diff
@@ -402,3 +402,13 @@ def test_shorthand_contains_dot() -> None:
     )
 
     assert actual == 'div(id="a", class_="w-[50.0625rem]")'
+
+
+def test_del_tag_is_replaced_with_del_() -> None:
+    actual = html2htpy(
+        "<div><del>deleted</del></div>",
+        shorthand_id_class=True,
+        import_mode="yes",
+    )
+
+    assert actual == 'from htpy import del_, div\ndiv[del_["deleted"]]'
```

Comment:
issue: htpy itself does not handle the trailing underscore properly. 😆  Did not consider that before.

```py
>>> from htpy import del_
>>> print(del_)
<del_></del_>
```


suggestion: We should also deal with the trailing underscores in the element names.  We need to do `name.removesuffix("_")` in `htpy/__init__.py` / `_get_element()` to remove the trailing `_`.

Reply:
Done!